arch.mk

```diff
@@ -2,18 +2,20 @@
 
 # check for FASTMATH or SP_MATH
 ifeq ($(SPMATH),1)
-  MATH_OBJS:=./lib/wolfssl/wolfcrypt/src/sp_int.o
+  MATH_OBJS:=./lib/wolfssl/wolfcrypt/src/sp_int.o ./lib/wolfssl/wolfcrypt/src/sp_c32.o
```

Comment:
This breaks compilation with `SIGN=ECC256 NO_ASM=1` on ARM (duplicate object). Also, we should not include sp_c32.o when compiling with assembly optimization (i.e. `NO_ASM=0`, which is the default for ECC). If you want to add sp_c32.o for RISC-V, please put it under `ifeq ($(ARCH),RISCV)`